crates/primitives/src/receipt.rs

```diff
@@ -14,6 +14,11 @@ use derive_more::{DerefMut, From, IntoIterator};
 use reth_codecs::{Compact, CompactZstd};
 use serde::{Deserialize, Serialize};
 
+pub trait Encodable2718: Encodable {
+    fn encode_envelope(&self, out: &mut dyn BufMut);
+    fn envelope_length(&self) -> usize;
+}
```

Comment:
this trait already exists, and we don't need to redefine it

it's

https://github.com/alloy-rs/alloy/blob/440ddcb872a47105ec18e470c8750258d863a610/crates/eips/src/eip2718.rs#L181-L181

Reply:
yeah, thank you for review, i just update code